crates/uv/tests/run.rs

```diff
@@ -171,3 +171,47 @@ fn run_args() -> Result<()> {
 
     Ok(())
 }
+
+/// Run a PEP 723-compatible script. The script should take precedence over the workspace
+/// dependencies.
+#[test]
+fn run_script() -> Result<()> {
```

Comment:
Because I don't think there's coverage for this, can you add a test for running a non PEP 723 script?

Reply:
Done!